packages/web/src/pages/profile-page/components/desktop/ProfilePage.tsx

```diff
@@ -171,27 +173,132 @@ export type ProfilePageProps = {
   onCloseUnmuteUserConfirmationModal: () => void
 }
 
+const PlaylistTab = ({
```

Comment:
Don't mind the wet code, but this file is a little long. Maybe can put these elsewhere or make a component? ok to leave it though

Reply:
i want to completely rewrite this file when we get rid of providers, so def a TODO